twitter/slackUpload.py

```diff
@@ -1,7 +1,7 @@
 import os
 import json
 import requests
-from twitter.config import (db, SLACK_WEBHOOK)
+from config import (db, SLACK_WEBHOOK)
```

Comment:
The `db` variable has not been defined in the `config` module. You will most likely have to create a new file to initialize it.

Reply:
yeah actually i added into th config module but at the end i didn't commit the changes .
i will change it now!